test/test_integration_cli.py

```diff
@@ -118,40 +126,35 @@ def generate_policy_string(self):
         variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
 
         policy = \
-"""
-- !variable
-  id: {variable_1}
-- !variable
-  id: {variable_2}
-- !variable
-  id: {variable_3}
-"""
+            """
+            - !variable
+              id: {variable_1}
+            - !variable
+              id: {variable_2}
+            - !variable
+              id: {variable_3}
+            """
 
         dynamic_policy = policy.format(variable_1=variable_1,
                                        variable_2=variable_2,
                                        variable_3=variable_3)
 
         return (dynamic_policy, [variable_1, variable_2, variable_3])
 
-
     # *************** TESTS ***************
     '''
     A non-existent policy file will return FileNotFound error message
     '''
+
     @integration_test
     def test_apply_policy_raises_file_not_exists_error(self):
-        output = invoke_cli(self, self.cli_auth_params,
-            ['policy', 'apply', 'root', 'somepolicy.yml'], exit_code=1)
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['policy', 'apply', 'root', 'somepolicy.yml'], exit_code=1)
         self.assertRegex(output, "Error: No such file or directory:")
 
     '''
     A non-existent variable file will return a 404 Not Found error message
     '''
-    @integration_test
-    def test_unknown_secret_raises_not_found_error(self):
```

Comment:
why was this removed 🙃

Reply:
I thought this test should move to a different suite but it's ok